pkg/kubelet/metrics/metrics.go

```diff
@@ -917,6 +922,40 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAOperationsDuration tracks the duration of the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           DRAOperationsDurationKey,
+			Help:           "Duration in seconds of the DRA operations (PrepareResources and UnprepareResources).",
+			Buckets:        metrics.DefBuckets,
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"operation_name"},
+	)
+
+	// DRAOperationsErrorsTotal is a counter that tracks the number of errors in the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsErrorsTotal = metrics.NewCounterVec(
```

Comment:
this will not be needed if isError dimension is on duration metric

Reply:
I think having a separate Count type metric for the total number of errors would make it easier for users to use for visualization and alerting since it's common practice to expose such metrics. Without a total error count metric, users would likely need to put more thought into how to achieve this.

Reply:
Wouldn't the `_count` metric be automatically available in Prometheus? Just add the _count to the histogram metric name?

Reply:
Yes, you're right. In the case of Histogram type metrics, a `_count` metric is automatically generated, so the same thing can be achieved if there is a label indicating whether it's an error. Although it might still cause some confusion for users since it works differently from the other existing metrics, reducing the number of metrics is a good thing as it decreases the amount of data.

Reply:
@SergeyKanzhelev @toVersus Thank you for the review! I'll drop this metric and add `isError` dimension to the `DRAOperationsDuration`.

Reply:
done, PTAL again, thanks!